scania-digital-design-system/tegel
# feat(text field): prefix and component name change

**Describe pull-request**  
Update the component name and updated the prefix.

**Solving issue**  
Fixes: 

**How to test**  
1. Go to storybook
2. Check in Components -> Text Field
3. Check the component API and styles.
